### PR TITLE
fix: Bug In The Link - MathMagicians-Calculator

### PR DESCRIPTION
**Pull request by mdibuhossain**
fixed #[153](https://github.com/Azordev/various-tasks/issues/153)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ cd mathmagicians-calculator
 npm i && npm start
 ```
 
-Then open [http://localhost:3000/](http://localhost:3000/) to see the app.
+Then open [https://math-magicians-calculator.herokuapp.com/](https://math-magicians-calculator.herokuapp.com/) to see the app.
```
Comment (Israel-Laguan) [inline README.md]: I think as the instructions are about development, what a dev would need is to open `localhost:3000`

I suggest substituting for this string `http://localhost:3000/`

Comment (mdibuhossain) [reply README.md]: Do you suggest like this?

[http://localhost:3000/](https://math-magicians-calculator.herokuapp.com/)

File: README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ cd mathmagicians-calculator
 npm i && npm start
 ```
 
-Then open [http://localhost:3000/](http://localhost:3000/) to see the app.
+Then open [http://localhost:3000/](https://math-magicians-calculator.herokuapp.com/) to see the app.
```
Comment (Israel-Laguan) [inline README.md]: Please try this one
```suggestion
Then open `http://localhost:3000/` to see the app.
```

Comment (mdibuhossain) [reply README.md]: The task was when I will click on `http://localhost:3000/` this link it will open in a new tab and visit that link.
Is that correct?

Comment (Israel-Laguan) [reply README.md]: Please ask Ana Piris

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**